setup.cfg

```diff
@@ -1,3 +1,48 @@
+[metadata]
+name = nbqa
+version = 0.3.6
+description = Run any standard Python code quality tool on a Jupyter Notebook
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/nbQA-dev/nbQA
+author = Marco Gorelli, Girish Pasupathy, Sebastian Weigand
+license = MIT
+license_file = LICENSE
+classifiers =
+    Development Status :: 4 - Beta
+    Intended Audience :: Developers
+    License :: OSI Approved :: MIT License
+    Natural Language :: English
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+
+[options]
+packages = find:
+py_modules = nbqa
+install_requires =
+    toml
+    importlib_metadata;python_version < '3.8'
+python_requires = >=3.6
+
+[options.entry_points]
+console_scripts =
+    nbqa = nbqa.__main__:main
+
+[options.package_data]
+nbqa =
+    config_files/default_config.toml
+
+[options.packages.find]
+exclude =
+    tests*
+
+[bdist_wheel]
+universal = True
+
 [bumpversion]
 current_version = 0.3.6
 
```

Comment:
This can be removed one or the other way (see https://github.com/nbQA-dev/nbQA/pull/403#issuecomment-719933624)